modules/gitops-installing-argocd-agent-using-helm.adoc

```diff
@@ -0,0 +1,149 @@
+// Module included in the following assembly:
+//
+// * argocd-agent-installation/gitops-installing-argocd-agent-using-helm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-installing-argocd-agent-using-helm_{context}"]
+= Installing the Argo CD Agent by using a Helm chart
+
+Use the Helm chart provided by the Argo CD Agent project to install the Agent component on the workload cluster. Argo CD Agent supports two modes for installation, *managed* and *autonomous*.
+
+.Procedure
+
+. Install the Managed Agent.
+In *managed* mode, the Agent must communicate with the spoke Redis server. However, the default network policy created by the {gitops-title} Operator does not allow this. To enable communication, you must create a custom network policy.
+
+.. Create a custom network policy YAML file:
++
+[source,yaml]
+----
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: custom-agent-redis-network-policy
+spec:
+  podSelector:
+    matchLabels:
+      app.kubernetes.io/name: openshift-gitops-redis
+  ingress:
+    - ports:
+        - protocol: TCP
+          port: 6379
+      from:
+        - podSelector:
+            matchLabels:
+              app.kubernetes.io/name: argocd-agent-agent
+  policyTypes:
+    - Ingress
+----
+
+.. Apply the network policy by using the following command:
++
+[source,terminal]
+----
+$ oc apply -f network-policy.yaml -n "<AGENT_NAMESPACE>" --context "<AGENT_CONTEXT>"
+----
+
+.. After applying the network policy, install the managed Agent using Helm by using the following command:
++
+[source,terminal]
+----
+$ helm install argocd-agent "<HELM_REPO>" \
```

Comment:
Question for Jayendra, where do we tell the user what value to put in this field?

Reply:
If we know what repo name would be given, we can add that in advance here to avoid rework on docs.